src/main/scala/is/hail/annotations/ReadRowsRDD.scala

```diff
@@ -264,7 +322,14 @@ class Encoder(out: DataOutputStream) {
   }
 }
 
-class RichRDDUnsafeRow(val rdd: RDD[UnsafeRow]) extends AnyVal {
+class RichRDDRegionValue(val rdd: RDD[RegionValue]) extends AnyVal {
+  def writeInt(out: OutputStream, i: Int) {
+    out.write(i & 0xFF)
+    out.write((i >> 8) & 0xFF)
+    out.write((i >> 16) & 0xFF)
+    out.write((i >> 24) & 0xFF)
```

Comment:
In this case, the & is unnecessary, right? We should leave it, I just want to make sure I understand what's going on.